src/style.css

```diff
@@ -1,3 +1,79 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4 px rgba(0 0 0 0.05);
+}
+
+.header__link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.header__logo {
+  height: 100%;
+  width: 100%;
+}
+
+.nav {
+  display: block;
```

Comment:
The `.nav` class has a `display: block;` property, which is the default display value for the `nav` element. This line can be removed as it is redundant.